table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -211,8 +212,12 @@ public SyncStatus sync(JournalContext context) throws IOException {
 
     if (Files.exists(Paths.get(mConfigPath))) {
       ObjectMapper mapper = new ObjectMapper();
-      DbConfig dbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
-      mBypassSet.addAll(dbConfig.getBypassSet());
+      try {
+        mDbConfig = mapper.readValue(new File(mConfigPath), DbConfig.class);
+      } catch (JsonProcessingException e) {
+        LOG.error(String.format("failed to deserialize UDB config file %s, stays unsynced",
+            mConfigPath), e);
```

Comment:
```suggestion
        LOG.error("Failed to deserialize UDB config file {}, stays unsynced", mConfigPath, e);
```

Check logging convention in Alluxio,
https://docs.alluxio.io/os/user/edge/en/contributor/Code-Conventions.html#best-practice

Reply:
Thanks. I also changed the log level to warn.